docs/source/contents/4_draft_mapping.rst

```diff
@@ -198,12 +199,12 @@ Justification
 ^^^^^^^^^^^^^
 
 OCLC identifiers could be useful if these materials are ever shared with HathiTrust, as this value is a requirement for
-submission. Only one property, dbpedia:oclc, was identified to use and it aligns with our philosophy guidelines.
+submission. Only one property, `dbpedia:oclc`, was identified to use and it aligns with our philosophy guidelines.
```

Comment:
It looks like @mlhale7 has decided to go with `dbo` instead of `dbpedia`. Not sure where is best, but I'd just change this here.

Reply:
Thanks for catching this - yes, this needs to change to dbo. I changed it in the table, but not the text.

Reply:
Thanks for catching this.